### ISSUE TITLE
RTL Languages

### ISSUE THREAD

**Issue by mohkamfer** [Aluxian/Facebook-Messenger-Desktop#550]
The **Type a message...** text box doesn't change its direction to RTL when typing with an RTL language.
In my case, it's Arabic.

**Comment by Sytten** [comment]
Is it working fine when you type in directly in messenger.com?

**Comment by mohkamfer** [comment]
> Is it working fine when you type in directly in messenger.com?

Nope, but I believe this isn't a problem, right?

**Comment by Sytten** [comment]
Yes it is since the software is only a wrapper around the website ;)

**Comment by Sytten** [comment]
In the old version, when you right clicked you could change the writing direction. We will see if its possible with electron (it should).

**Comment by mohkamfer** [comment]
I honestly believe pure JavaScript, aside from Electron, can handle this... It just needs to check if the first character inputted is an ASCII of RTL languages, if so it'll set the direction of the textbox to RTL. Otherwise, it won't touch the textbox.

I might be wrong as I  have never dealt with Electron and actually just Googled it to learn about it.

**Comment by Sytten** [comment]
@Aluxian kind of important
I believe electron can handle that by itself, must be a setting or something
